cfg/VideoStream.cfg

```diff
@@ -29,6 +29,8 @@ gen.add("hue", double_t, LEVEL.RUNNING, "Target hue", 0.5, 0.0, 1.0)
 gen.add("saturation", double_t, LEVEL.RUNNING, "Target saturation", 0.64, 0.0, 1.0)
 gen.add("auto_exposure", bool_t, LEVEL.RUNNING, "Target auto exposure", True)
 gen.add("exposure", double_t, LEVEL.RUNNING, "Target exposure", 0.5, 0.0, 1.0)
+gen.add("auto_focus", bool_t, LEVEL.RUNNING, "Target auto focus", True)
```

Comment:
What happens if we use camera without auto focus feature?
Is `cap->set(cv::CAP_PROP_AUTOFOCUS, 1);` just ignored?

Reply:
@furushchev The camera I have here without autofocus works fine.